src/Microsoft.Diagnostics.Monitoring.EventPipe/Configuration/LoggingSourceConfiguration.cs

```diff
@@ -11,26 +12,33 @@ namespace Microsoft.Diagnostics.Monitoring.EventPipe
 {
     public class LoggingSourceConfiguration : MonitoringSourceConfiguration
     {
+        private const string UseAppFilters = "UseAppFilters";
+
         private readonly LogLevel _level;
+        private readonly bool _useAppFilter;
 
-        public LoggingSourceConfiguration(LogLevel level = LogLevel.Debug)
+        public LoggingSourceConfiguration(LogLevel level = LogLevel.Debug, bool useAppFilter = false)
         {
             _level = level;
+            _useAppFilter = useAppFilter;
```

Comment:
nit: `_useAppFilters` field and `useAppFilters` parameter (note the 's' on the end)